# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/utils/PerformanceRecorder.h
# Commit: e85b278387fc
# Full Hash: e85b278387fc5259efead947f5069c9c2dcbffad
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-11 21:56:57
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 0fa64a630cc7 -r e85b278387fc dom/media/utils/PerformanceRecorder.h
--- a/dom/media/utils/PerformanceRecorder.h	Wed Jan 11 11:47:45 2023 +0000
+++ b/dom/media/utils/PerformanceRecorder.h	Wed Jan 11 11:47:45 2023 +0000
@@ -151,7 +151,7 @@
     MJPEG,
   };
 
-  CaptureStage(nsCString aSource, nsCString aTrackingId, int32_t aWidth,
+  CaptureStage(nsCString aSource, TrackingId aTrackingId, int32_t aWidth,
                int32_t aHeight, ImageType aImageType)
       : mSource(std::move(aSource)),
         mTrackingId(std::move(aTrackingId)),
@@ -165,10 +165,12 @@
   }
 
   nsCString mSource;
-  nsCString mTrackingId;
+  TrackingId mTrackingId;
   int32_t mWidth;
   int32_t mHeight;
   ImageType mImageType;
+
+ private:
   mutable Maybe<nsCString> mName;
 };
 